### PR TITLE
Fix Copy Calls in HCMI

### PR DESCRIPTION
**Pull request by jjacobson95**
This aligns HCMI copy calls with the other datasets.

This addresses issue #268.

### CODE REVIEW

**Comment by jjacobson95** [comment]
@sgosline this should be ready to merge. I'll start build version 0.2.0 once merged.

File: build/hcmi/02-getHCMIData.py [hunk 1]
```diff
@@ -474,15 +474,15 @@ def get_copy_call(a):
 
         if math.isnan(a):
             return float('nan')
-        
-        a_val = math.log2(float(a)+0.000001) ###this should not be exponent, should be log!!! 2**float(a)
-        if a_val < 0.0: #0.5210507:
+
+        a_val = math.log2(float(a)+0.000001)
```
Comment (sgosline) [inline build/hcmi/02-getHCMIData.py:478]: please double check this. we should not be doing log space unless they somehow threw in an exponent. values shoud range between 0 and 2 (or more in some cases)

Comment (jjacobson95) [reply build/hcmi/02-getHCMIData.py:478]: Copy numbers here are all integer values from 0-214 (with log) but primarily centered around 2 which gets converted to 1 and called as diploid.

Here is a distribution of the values (with log) taken from the most recently generated [version](https://figshare.com/account/projects/189342/articles/28038896?file=51224342).

---

**All counts across all samples:**
image


**All counts across all samples for Copy Numbers 0-20:**
image


**Copy Call Counts Table**
| Count        | Copy Call      |
|-----------|-----------|
| 7622238   | amp       |
| 14048     | deep del  |
| 14310695  | diploid   |
| 5030752   | gain      |
| 1673926   | het loss  |

**Comment by sgosline** [comment]
I'm not sure I understand what is happening. The top plot CNV peaks at 0. I believe these are log2 ratios (based on a brief google search of cptac papers). However, when you calculate 2**CNV you should get a peak at 1, not 0.

**Comment by jjacobson95** [comment]
Sorry, yes you are correct. I was performing the wrong calculation in the code.

**Updated plot** (lines are weird, but yes this is centered around 1).
image


Still, we are are doing copy calls on Untransformed* for CPTAC but doing copy calls for Transformed (log2) data on HCMI.

Untransformed* because CPTAC transformed their data using log2 and we undo it.

**Pull request merged**